rust/src/driver.rs

```diff
@@ -232,13 +247,13 @@ impl TypeDBDriver {
                 Ok(server_connection) => match server_connection.servers_all() {
                     Ok(servers) => return Ok(servers.into_iter().collect()),
                     Err(Error::Connection(
-                        ConnectionError::ServerConnectionFailedStatusError { .. } | ConnectionError::ConnectionFailed,
-                    )) => (),
+                            ConnectionError::ServerConnectionFailedStatusError { .. } | ConnectionError::ConnectionFailed,
```

Comment:
Should be autoformatter and I get its idea, but this indentation looks incorrect